src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -28,14 +28,15 @@ private static void FreeSymbolMap(MultiDictionary<string, ISymbol> symbolMap)
             Project project, CancellationToken cancellationToken)
         {
             var compilation = await project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
+            var checksum = await project.State.GetChecksumAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
this is fine, but I need to fix one thing monday to make it work properly. this checksum contains document checksum as well which contains documentId which is not same across session. I need the checksum to be project filepath + document filepath if exist, otherwise, documentId as before to make it same across sessions.